modules/creating-scaling-machine-sets-compute-nodes-networking.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+// UPI
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-scaling-machine-sets-compute-nodes-networking.adoc_{context}"]
+= Optional: Scaling each machine set to compute nodes
+
+To apply a customized `br-ex` bridge configuration to all compute nodes in your {product-title} cluster, you must edit your `MachineConfig` custom resource definition (CRD) and modify its roles. Additionally, you must create a `BareMetalHost` CRD that defines information for your bare-metal machine, such as hostname, credentials, and so on.
```

Comment:
Just checking that this indeed is about modifying the _CRD_ itself, and not an individual _CR_ instance. The `oc edit mc` command later made me wonder (since that looks more like modifying an individual `Kind` type), but maybe things work different for MachineConfigs. 🤷 

If this should actually be CR, then:
```suggestion
To apply a customized `br-ex` bridge configuration to all compute nodes in your {product-title} cluster, you must edit your `MachineConfig` custom resource (CR) and modify its roles. Additionally, you must create a `BareMetalHost` CR that defines information for your bare-metal machine, such as hostname, credentials, and so on.
```
And also adjust the later instances of "CRD" in this module to "CR" to match.